plugins/scaffolder-backend-module-gitlab/README.md

```diff
@@ -0,0 +1,118 @@
+# scaffolder-backend-module-gitlab
```

Comment:
Add example how to add the actions to the scaffolder plugin

Reply:
done